### DISCUSSION TITLE
How to get all chord types using RomanNumeral.get and a Key signature

### DISCUSSION THREAD

**Discussion by ReneDiazDiversity** [tonaljs/tonal#295] [Q&A]
A problem I am having. I was going to use the roman numerals to get chords. 
Example
Key "C", Imaj, iimin7, bIIImin7b9, viimin7b5b9 returns: C, Dmin7, Ebmin7b9, Bmin7b5b9
or
 getMyChord("C", "II9"); // returns "Dm9"
or
or getMyChord("C", "IIm9"); // returns "Dm9" 

But it seems the RomanNumeral.get only does 7 chords not even triads. 
Then I was going to use the Key.majorKey which has amazing features but that also seems to only pull 7 chords 

Again I basically want to use a key signature and then use roman numerals and chord types to call up chords and play them. for example in the key of C IMaj7 IVMaj7 and V7 for C Maj7 FMaj7 and G7 

And also have it play the chords as  arpeggios. But none of this is working.

Any help will be great. 

![image](https://user-images.githubusercontent.com/20298208/147713708-86ed1ed4-f30b-4e34-b14b-872c53bbeb7e.png)

![image](https://user-images.githubusercontent.com/20298208/147713721-1f0a6b4f-872a-4313-995a-eacaac7638f4.png)

![image](https://user-images.githubusercontent.com/20298208/147713735-71fbb25d-271e-4e96-a776-da36b14c2933.png)

**Comment by mrjacobbloom** [comment]
Does this help with the roman numerals part of your question? https://github.com/tonaljs/tonal/discussions/296#discussioncomment-1890615

**Comment by ReneDiazDiversity** [comment]
It didn't work 😦
We tried to get the key, Key.majorKey("C")
https://github.com/tonaljs/tonal/tree/main/packages/key
and Scale
https://github.com/tonaljs/tonal/tree/main/packages/scale
and for the chord root we tried both below
intervals: ["1P", "2M", "3M", "4P", "5P", "6M", "7M"], Which I was hoping would work. 

and tried 
grades: ["I", "II", "III", "IV", "V", "VI", "VII"],


Intervals = 1P for C 
to play a C major9 chord
  {
                              type: 'chord',
                              notation: 'note',
                              chord: {
                                 type: 'maj9',
                                 intervals: '1P',
                              },

 gradess = I for C
to play a C major9 chord
  {
                              type: 'chord',
                              notation: 'note',
                              chord: {
                                 type: 'maj9',
                                 gradess: 'I',
                              },

None of them worked 😦

**Reply by ReneDiazDiversity** [reply]
It didn't work 😦
We tried to get the key, Key.majorKey("C")
https://github.com/tonaljs/tonal/tree/main/packages/key
and Scale
https://github.com/tonaljs/tonal/tree/main/packages/scale
and for the chord root we tried both below
intervals: ["1P", "2M", "3M", "4P", "5P", "6M", "7M"], Which I was hoping would work. 

and tried 
grades: ["I", "II", "III", "IV", "V", "VI", "VII"],


Intervals = 1P for C 
to play a C major9 chord
  {
                              type: 'chord',
                              notation: 'note',
                              chord: {
                                 type: 'maj9',
                                 intervals: '1P',
                              },

 gradess = I for C
to play a C major9 chord
  {
                              type: 'chord',
                              notation: 'note',
                              chord: {
                                 type: 'maj9',
                                 gradess: 'I',
                              },

None of them worked 😦
